api/tests/test_user_feedback_api.py

```diff
@@ -0,0 +1,89 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+from unittest.mock import patch
+
+from ..models import User, UserFeedback
+
+client = APIClient()
+
+
+class UserFeedbackModelTest(TestCase):
+    """ Tests for the UserFeedback endpoint"""
+
+    def setUp(self):
+        self.user = User.objects.create(
+            email='test4@site.com', cohort=20,
+            slack_handle='@test_user4', password='devpassword'
+        )
+
+        UserFeedback.objects.create(reported_by="X.Y@andela.com",
+                                    message="This is feedback",
+                                    report_type="bug")
+
+        self.feedback_url = '/api/v1/user-feedback/'
+        self.token_user = 'testtoken'
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_can_post_feedback(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.user.email}
+        data = {
+            "reported_by": "test4@site.com",
```

Comment:
we shouldn't be having `reported_by` sent to the API endpoint.

Reply:
It will come from the token ?

Reply:
i believe you're setting it [here](https://github.com/AndelaOSP/art-backend/pull/55/files#diff-265524153427c5c3f0dd9ae087f16a4bR72) :)